### ISSUE TITLE
concise.css/src/_atgrid.scss [container] 

### ISSUE THREAD

**Issue by FredHall** [ConciseCSS/concise.css#262]

**Comment by FredHall** [comment]
James,

The previous "issue" I opened and you accepted on this topic has a problem. Sorry.

If at the end of a container there is something similar to a dropdown menu with hidden items that extend beyond the container, a double scroll bar is displayed. See attached screenshot. The screenshots show Firefox, Chrome and Safari under OS-X.

I stumbled across a very simple and Concise solution. 
1) Remove the "overflow: auto" we added to _atgrid.scss
2) In concise.css/src/_base.scss 
:root {
--  background-color: getColor(background, body);

++ body { 
++   background-color: getColor(background, body);
++ }

I am uncertain why this fix works, it makes no sense to me, but it does. 

I believe this change is safe for the current version of Concise CSS.

![container-overflow-auto](https://cloud.githubusercontent.com/assets/24819365/22001840/c6755960-dc04-11e6-8a42-5e55362c0675.png)
![root-background-color](https://cloud.githubusercontent.com/assets/24819365/22001841/c85418fc-dc04-11e6-89b0-cce4a84b6430.png)
![no-root-background-color](https://cloud.githubusercontent.com/assets/24819365/22001848/d18563ea-dc04-11e6-89a2-2dfb2bad7d80.png)

**Comment by jameskolce** [comment]
Hi @FredHall 

I also noticed that changing the background color from `:root` to `body` solved the problem, but because your solution also worked for me, I used that one, I didn't know it also created another scrollbar. 

So I’ll add this change, but I’m not quite sure if it’s safe for a patch version because some users might be doing unexpected things so we can’t assume. What I’m going to do, is to remove the `overflow: auto` and change the background rule in a next major version.

But don’t worry, that won’t take too long, I have some plans for that as I mentioned in the last email :)
